### ISSUE TITLE
OpenReferral only permits taxonomies associated with services

### ISSUE THREAD

**Issue by MikeThacker1** [esd-org-uk/human-services#2]
OpenReferral has service_taxonomy to associate taxonomy terms with services, but no provision for taxonomies associated with other objects, such as organizations.

The proposed LGA extension therefore adds link_taxonomy with a link_type that can be populated with one of:

- organisation
- eligibility
- coverage
- cost_option
- taxonomy

This is a potentially confusing approach but I'm not sure it can be avoided without breaking the original OpenReferral standard.

**Comment by greggish** [comment]
Can you clarify what you mean by `coverage`? Is that like `service_area`?

And, the final bullet of `taxonomy`– does this suggest a taxonomy for these taxonomies? or is that referring to the existing `service_taxonomy` concept?

**Comment by MikeThacker1** [comment]
Yes, my understanding of the design is that `coverage` contains geographies that define service areas, eg the URI http://statistics.data.gov.uk/id/statistical-geography/E09000033 from the Office of National Statistics statistical geographies taxonomy. [The full ERD](https://raw.githubusercontent.com/esd-org-uk/human-services/master/Resources/AllERD.png) is here.

The final bullet of `taxonomy` represents a copy and paste error into the [extended tabular data package](https://raw.githubusercontent.com/esd-org-uk/human-services/master/SchemaGenerator/Generator/ExtendedDataPackage.json). Sorry. I've asked for that to be removed and the [documentation](http://htmlpreview.github.io/?https://github.com/esd-org-uk/human-services/blob/master/Schemas/documentation.html) regenerated.
